src/main/java/edu/kis/powp/jobs2d/drivers/DriverWrapper.java

```diff
@@ -0,0 +1,22 @@
+package edu.kis.powp.jobs2d.drivers;
+
+import edu.kis.powp.jobs2d.AbstractDriver;
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.features.DriverFeature;
+
+
+public class DriverWrapper extends AbstractDriver{
```

Comment:
The name does not indicate the purpose of the adapter or what was actually adapted.